cg/cli/workflow/microsalt/base.py

```diff
@@ -260,3 +266,82 @@ def start_available(context: click.Context, dry_run: bool):
 
     if exit_code:
         raise click.Abort
+
+
+@microsalt.command("upload-analysis-vogue")
+@OPTION_DRY_RUN
+@ARGUMENT_UNIQUE_IDENTIFIER
+@click.pass_context
+def upload_analysis_vogue(context: click.Context, unique_id: str, dry_run: bool) -> None:
+    """Upload the trending report for latest analysis of given case_id to Vogue"""
+
+    vogue_api = VogueAPI(context.obj)
+    microsalt_analysis_api: MicrosaltAnalysisAPI = context.obj["microsalt_analysis_api"]
+    case_obj = microsalt_analysis_api.db.family(unique_id)
+    if not case_obj or not case_obj.analyses:
+        LOG.error("No analysis available for %s", unique_id)
+        raise click.Abort
+
+    samples_string = ",".join(str(link_obj.sample.internal_id) for link_obj in case_obj.links)
+    microsalt_version = microsalt_analysis_api.get_microsalt_version()
+
+    if dry_run:
+        LOG.info(
+            "Would have loaded case %s, with samples %s, analyzed with pipeline version %s",
+            unique_id,
+            samples_string,
+            microsalt_version,
+        )
+        return
+
+    analysis_result_file = microsalt_analysis_api.hk.get_files(
+        bundle=unique_id, tags=["vogue"]
+    ).first()
+    if not analysis_result_file:
+        LOG.error("Vogue upload file not found in Housekeeper for case %s", unique_id)
+        raise click.Abort
+
+    vogue_load_args = {
+        "samples": samples_string,
+        "analysis_result_file": analysis_result_file.full_path,
+        "analysis_case_name": unique_id,
+        "analysis_type": "microsalt",
+        "analysis_workflow_name": "microsalt",
+        "analysis_workflow_version": microsalt_version,
+        "case_analysis_type": "microsalt",
+    }
+    vogue_api.load_bioinfo_raw(load_bioinfo_inputs=vogue_load_args)
+    vogue_api.load_bioinfo_process(load_bioinfo_inputs=vogue_load_args, cleanup_flag=False)
+    vogue_api.load_bioinfo_sample(load_bioinfo_inputs=vogue_load_args)
+    case_obj.analyses[0].uploaded_at = dt.datetime.now()
+    microsalt_analysis_api.db.commit()
+    LOG.info("Successfully uploaded latest analysis data for case %s to Vogue!", unique_id)
+
+
+@microsalt.command("upload-latest-analyses-vogue")
+@OPTION_DRY_RUN
+@click.pass_context
+def upload_vogue_latest(context: click.Context, dry_run: bool) -> None:
+    """Upload the trending reports for all un-uploaded latest analyses to Vogue"""
+
+    EXIT_CODE: int = EXIT_SUCCESS
+    microsalt_analysis_api: MicrosaltAnalysisAPI = context.obj["microsalt_analysis_api"]
+    latest_analyses = list(
```

Comment:
Could not that function return None?

Reply:
its a query, which can be empty, and then it turns into empty list